content/docs/next-release/operator.md

```diff
@@ -0,0 +1,645 @@
+
+
+
+|Up to version |apiVersion |CRD yaml
+| --- | --- | --- |
+|master |jaegertracing.io/v1 |https://github.com/jaegertracing/jaeger-operator/blob/master/deploy/crds/jaegertracing_v1_jaeger_crd.yaml[jaegertracing_v1_jaeger_crd.yaml]|
+|1.10.0 |io.jaegertracing/v1alpha1 |https://github.com/jaegertracing/jaeger-operator/blob/master/deploy/crds/io_v1alpha1_jaeger_crd.yaml[io_v1alpha1_jaeger_crd.yaml]|
+
+
+## Installing the Operator on Kubernetes
+
+The following instructions will create the `oberservability` namespace and intall the Jaeger operator.
+
+NOTE: Make sure your `kubectl` command is properly configured to talk to a valid Kubernetes cluster. If you don't have a cluster, you can create one locally using [`minikube`](https://kubernetes.io/docs/tasks/tools/install-minikube/).
+
+To install the operator, run:
+<!--TODO - Does Kubernetes have privileged users? Needs to be run as a system:admin on OKD/OpenShift.-->
+
+```bash
+kubectl create namespace observability # <1>
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing_v1_jaeger_crd.yaml # <2>
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/service_account.yaml
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role.yaml
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/role_binding.yaml
+kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/operator.yaml
+```
+<1> This creates the namespace used by default in the deployment files. If you want to install the Jaeger operator in a different namespace, you must edit the deployment files to change `observability` to the desired namespace value.
+
+<2> This installs the "Custom Resource Definition" for the `apiVersion: jaegertracing.io/v1`
+
+IMPORTANT: When using a Jaeger Operator up to v1.10.0, install the CRD file `io_v1alpha1_jaeger_crd.yaml` in addition to `jaegertracing_v1_jaeger_crd.yaml`. This is because up to that version, the `apiVersion` in use was `io.jaegertracing/v1alpha1`.
+
+At this point, there should be a `jaeger-operator` deployment available.  You can view it by running the following command:
+
+```bash
+$ kubectl get deployment jaeger-operator -n observability
+
+NAME              DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+jaeger-operator   1         1         1            1           48s
+```
+
+The operator is now ready to create Jaeger instances.
+<!-- TODO - Answer question - Which components are installed with a Jaeger instance?  Agent, Collector, Storage (Cassandra?  Elasticsearch?), Query, Ingestor?  What are the default values? -->
```

Comment:
This is part of the "deployment strategies". The operator decides which components to install, and how to run them, based on the strategy. For instance, "all in one" deploys the "all-in-one" image in a single pod, using in-memory by default. When setting "production", storage information becomes mandatory, as we can't use in-memory for that (no way to share the same memory space between two pods).